docs/topics/whatsnew-eap.md

```diff
@@ -800,12 +856,86 @@ keep in mind that you might encounter deprecation warnings or some new Gradle fe
 
 This version brings the following changes:
 
+* [New Gradle DSL for compiler options in multiplatform projects](#new-gradle-dsl-for-compiler-options-in-multiplatform-projects)
+* [New attribute to distinguish JVM and Android published libraries](#new-attribute-to-distinguish-jvm-and-android-published-libraries)
 * [Improved Gradle dependency handling for CInteropProcess in Kotlin/Native](#improved-gradle-dependency-handling-for-cinteropprocess-in-kotlin-native)
 * [Visibility changes in Gradle](#visibility-changes-in-gradle)
 * [New directory for Kotlin data in Gradle projects](#new-directory-for-kotlin-data-in-gradle-projects)
 * [Kotlin/Native compiler downloaded when needed](#kotlin-native-compiler-downloaded-when-needed)
 * [Deprecating old ways of defining compiler options](#deprecating-old-ways-of-defining-compiler-options)
 
+### New Gradle DSL for compiler options in multiplatform projects
+
+> This feature is [Experimental](components-stability.md#stability-levels-explained). It may be dropped or changed at any time.
+> Use it only for evaluation purposes. We would appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
+>
+{type="warning"}
+
+Prior to Kotlin %kotlinEapVersion%, configuring compiler options in a multiplatform project with Gradle was only possible
+at a low level, such as per task, compilation, or source set. To make it easier to configure compiler options more generally
+in your projects, Kotlin %kotlinEapVersion% comes with a new Gradle DSL.
+
+With this new DSL, you can configure compiler options at the extension level for all the targets and shared source sets
+like `commonMain` and at a target level for a specific target:
+
+```kotlin
```

Comment:
I feel that the example comes between two parts of the conceptual explanation (and that leads to "extension level" being explained later than it is mentioned). More natural sequence would be "... was only possible at low level → project configuration now has three layers → here's an example → here's nuance about conventions".

Reply:
I think this sequence would work for the actual documentation. Here, in What's new, we need to announce the news, the new DSL. The whole section starting with "3 layers" is additional information to explain how the overall picture looks like now, but it can be removed without losing the point of the announcement

Reply:
My main point is about the "extension level"; if the intended audience doesn't need the term introduced here, then it's all good.